hooks/pre_values/10-register_fusor_modules.rb

```diff
@@ -4,5 +4,5 @@
              {:manifest_name => 'plugin/fusor',  :dir_name => 'foreman'})
 
   # make sure foreman-tasks is enabled
-  kafo.module('foreman_plugin_tasks').enable
+  #kafo.module('foreman_plugin_tasks').enable
```

Comment:
Figure out why this doesn't work. It doesn't seem to be needed so maybe disable for devel installer?